editor/src/components/editor/editor-component.tsx

```diff
@@ -91,8 +91,10 @@ function pushProjectURLToBrowserHistory(
     // …but if it's forking, remove the fork param
     queryParams.delete(ForkSearchParamKey)
   }
-  // remove accessLevel param
-  queryParams.delete(AccessLevelParamKey)
+  // remove one-time creation params
+  ;[AccessLevelParamKey, CloneParamKey].forEach((key) => {
+    queryParams.delete(key)
+  })
```

Comment:
It feels pretty unnecessary to wrap this in a `forEach` rather than just calling the function twice explicitly

Reply:
well for now it's two params, but we may want to add more in the future